antwar.config.js

```diff
@@ -67,6 +67,8 @@ module.exports = {
           /^\.\/.*\.md$/
         );
       }, {
+        'code-splitting-import': '/guides/code-splitting-import',
```

Comment:
@simon04 yeah but didn't you remove the `code-splitting-import` guide? Shouldn't the redirect be to `/guides/code-splitting-async`?